sonic_platform_base/sonic_storage/storage_common.py

```diff
@@ -0,0 +1,63 @@
+#
+# storage_common.py
+#
+# Class for common implementation functions,
+# i.e. functions that are storage device type agnostic.
+#
+
+try:
+    import os
+    import sys
+    import psutil
+except ImportError as e:
+    raise ImportError (str(e) + "- required module not found")
+
+
+class StorageCommon(object):
+    def __init__(self, diskdev):
+        """
+        Constructor
+
+        Args:
+            Block device path for which we need to get information
+        """
+
+        self.storage_disk = os.path.basename(diskdev)
+        self.fsstats_reads = 0
```

Comment:
I like to "initialize" fsstats_reads and fsstats_writes to 0 so that if the subsequent try blocks catch an exception, I can stick with `return self.fsstats_reads\writes` instead of `return 0`. Thoughts?

Reply:
even with that, you don't need the member. Suggest:
```python
fsstats_reads = 0
try:
    fsstats_reads = xxx
except:
   # log something

return fsstats_reads
```

Reply:
addressed in latest commit